counsel.el

```diff
@@ -3291,10 +3330,9 @@ When non-nil, INITIAL-INPUT is the initial search pattern."
     (user-error "Current buffer is not visiting a file"))
   (counsel-require-program counsel-grep-base-command)
   (setq counsel-grep-command
-        (format counsel-grep-base-command
-                "%s" (shell-quote-argument
-                      (file-name-nondirectory
-                       buffer-file-name))))
+        (counsel--format counsel-grep-base-command
+                         "%s" (shell-quote-argument (file-name-nondirectory
+                                                     buffer-file-name))))
```

Comment:
Isn't the shell quoting unnecessary in the case that `counsel-git-grep` is run as a subprocess rather than as a shell command?